src/site/content/en/blog/iframe-lazy-loading/index.md

```diff
@@ -73,12 +73,6 @@ images until users scroll near them. `loading` supports three values:
 
 *   `lazy`: is a good candidate for lazy-loading.
 *   `eager`: is not a good candidate for lazy-loading. Load right away.
-*   `auto`: browser will determine whether or not to lazily load.
-
-{% Aside %}
-`auto` is currently a non-standard value, but is the default in Chrome today.
-Chrome intends on bringing a proposal for this value to the standards table.
-{% endAside %}
```

Comment:
I think that removing the sections on `auto` for now makes sense. We're evaluating a few proposals for automatically lazy-loading offscreen content such as iframes by default, which may mean coming back to this value at some point but until there's something concrete there, this sgtm.